internal/components/discovery/testdata/icp-test-cr.yaml

```diff
@@ -4,177 +4,41 @@ metadata:
   name: cp-v115x
   namespace: istio-system
 spec:
+  sidecarInjector:
+    templates:
+      sidecar: |
+        spec:
+          containers:
+          - name: istio-proxy
+            env:
+            - name: TEMPLATE
+              value: sidecar
+      gateway: |
+        spec:
+          containers:
+          - name: istio-proxy
+            env:
+            - name: TEMPLATE
+              value: gateway
+      customTemplates:
+        - name: "custom1"
+          template: |
+            spec:
+              containers:
+              - name: istio-proxy
+                env:
+                - name: TEMPLATE
+                  value: custom1
+        - name: "custom2"
+          template: |
+            spec:
+              containers:
+              - name: istio-proxy
+                env:
+                - name: TEMPLATE
+                  value: custom2
   meshID: mesh1
   clusterID: demo-cluster1
   version: "1.15.3"
   mode: ACTIVE
   networkName: network1
-  containerImageConfiguration:
```

Comment:
Is everything below this deliberately deleted?

Reply:
I guess CI broke because of this.

Reply:
fixed